services/components/lifecycle/bot_logger.go

```diff
@@ -0,0 +1,116 @@
+package lifecycle
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/forta-network/forta-core-go/security"
+	"github.com/forta-network/forta-core-go/clients/agentlogs"	
+	"github.com/forta-network/forta-node/clients"
+	"github.com/forta-network/forta-node/clients/docker"
+	"github.com/forta-network/forta-node/services/components/containers"
+	log "github.com/sirupsen/logrus"
+)
+
+// BotLogger manages bots logging.
+type BotLogger interface {
+	SendBotLogs(ctx context.Context) error
+}
+
+type SupervisorServiceConfig struct {
+	Key                *keystore.Key
+}
```

Comment:
Not 100% about this, there is an existing `SupervisorServiceConfig` that cannot be imported due to cycle imports, should we move this to another shared package?